client/python/cryoet_data_portal/src/cryoet_data_portal/_models.py

```diff
@@ -253,58 +246,58 @@ def download_everything(self, dest_path: Optional[str] = None):
+        id (int): Numeric identifier for this tomogram (this may change!)
         is_canonical (bool): Is this tomogram considered the canonical tomogram for the run experiment? True=Yes
         key_photo_thumbnail_url (str): URL for the thumbnail of key photo
         key_photo_url (str): URL for the key photo
         name (str): Short name for this tomogram
         neuroglancer_config (str): the compact json of neuroglancer config
         offset_x (int): x offset data relative to the canonical tomogram in pixels
         offset_y (int): y offset data relative to the canonical tomogram in pixels
-        offset_z (int):  z offset data relative to the canonical tomogram in pixels
+        offset_z (int): z offset data relative to the canonical tomogram in pixels
         processing (str): Describe additional processing used to derive the tomogram
         processing_software (str): Processing software used to derive the tomogram
-        reconstruction_method (str):Describe reconstruction method (Weighted back-projection, SART, SIRT)
-        reconstruction_software (str):Name of software used for reconstruction
-        s3_mrc_scale0 (str):S3 path to this tomogram in MRC format (no scaling)
-        s3_omezarr_dir (str):S3 path to this tomogram in multiscale OME-Zarr format
-        scale0_dimensions (str):comma separated x,y,z dimensions of the unscaled tomogram
-        scale1_dimensions (str):comma separated x,y,z dimensions of the scale1 tomogram
-        scale2_dimensions (str):comma separated x,y,z dimensions of the scale2 tomogram
+        reconstruction_method (str): Describe reconstruction method (Weighted backprojection, SART, SIRT)
+        reconstruction_software (str): Name of software used for reconstruction
+        s3_mrc_scale0 (str): S3 path to this tomogram in MRC format (no scaling)
+        s3_omezarr_dir (str): S3 path to the this multiscale omezarr tomogram
+        scale0_dimensions (str): comma separated x,y,z dimensions of the unscaled tomogram
+        scale1_dimensions (str): comma separated x,y,z dimensions of the scale1 tomogram
+        scale2_dimensions (str): comma separated x,y,z dimensions of the scale2 tomogram
         size_x (int): Number of pixels in the 3D data fast axis
         size_y (int): Number of pixels in the 3D data medium axis
-        size_z (int):  Number of pixels in the 3D data slow axis.  This is the image projection direction at zero stage tilt
+        size_z (int): Number of pixels in the 3D data slow axis.  This is the image projection direction at zero stage tilt
         tomogram_version (str): Version of tomogram using the same software and post-processing. Version of tomogram using the same software and post-processing. This will be presented as the latest version
-        tomogram_voxel_spacing (TomogramVoxelSpacing): An object relationship with a specific voxel spacing for this experiment run
-        type (str):  Tomogram purpose (ex: CANONICAL)
```

Comment:
This got lost in the generation process and should be fixed here.

We could consider defining a string enum to handle enum fields, though that's probably for another issue.

Reply:
Now fixed. The issue was that this an object type field, but not a model, which confused the current code generator logic. I included a workaround to specifically parse fields of model types, then handle everything else as a scalar, but that might be fragile.